splunk-otel-android/src/main/java/com/splunk/rum/internal/opentelemetry/OpenTelemetryRumBuilder.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.rum.internal.opentelemetry;
+
+import android.app.Application;
+import io.opentelemetry.sdk.metrics.SdkMeterProviderBuilder;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProviderBuilder;
+import java.time.Duration;
+import java.util.function.BiFunction;
+
+/**
+ * A builder of {@link OpenTelemetryRum}. It enabled configuring the OpenTelemetry SDK and disabling
+ * built-in Android instrumentations.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+public interface OpenTelemetryRumBuilder {
```

Comment:
Why interface? Do we need that yet? Are we expecting there to be more than one implementation of this? Are we overthinking or being too purist about sdk/api separation? In fact, I wonder if just starting with baking an OTel RUM SDK and then factoring APIs out as needed might have a better impact.....curious what you think.

Reply:
I simply made it an interface to illustrate the API better 😄 
I was thinking of changing this into a class when I get to the implementation part.

> In fact, I wonder if just starting with baking an OTel RUM SDK and then factoring APIs out as needed might have a better impact.....curious what you think.

Yeah, that might be a good idea. I think once I merge all outstanding PRs I'll create a very minimal implementation of `OpenTelemetryRum` that does not contain any instrumentations, just SDK setup and session id -- and then move instrumentations one by one in later PRs. WDYT?

Reply:
Yeah that sounds like a plan.